tools/phenotype_annotation/README.md

```diff
@@ -0,0 +1,78 @@
+Phenotype Annotation
+=====================================
+The script retrieves phenotype annotations from Ensembl for a given list of genes OR phenotypes OR rsIDs OR VCF file OR VEP output file.
+
+##### Table of contents
+* [Usage](#usage)
+* [Examples](#examples)
+---
+<a name="usage"></a>
+### Usage
+The script is a wrapper of the Ensembl Variation API to retrieve the phenotype annotations.
+
+#### Running the script
+```bash
+./phenotype_annotation -input list_genes.txt -inputType gene
```

Comment:
It is odd that inputType is camel case and the rest of the options are snake case

Reply:
Good point, I updated it to input_type.